mvvm/src/main/java/com/thefuntasty/mvvm/delegate/ViewModelFragmentDelegate.kt

```diff
@@ -0,0 +1,21 @@
+package com.thefuntasty.mvvm.delegate
+
+import androidx.fragment.app.Fragment
+import com.thefuntasty.mvvm.BaseView
+import com.thefuntasty.mvvm.BaseViewModel
+import com.thefuntasty.mvvm.ViewModelView
+import com.thefuntasty.mvvm.ViewState
+import com.thefuntasty.mvvm.event.Event
+import kotlin.reflect.KClass
+
+@Suppress("UNCHECKED_CAST")
+interface ViewModelFragmentDelegate<VM : BaseViewModel<VS>, VS : ViewState> : ViewModelView<VM>, BaseView {
+
+    fun initViewModel(fragment: Fragment): VM {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/mvvm-android/blob/4a02c207953ea3674887f7e1a945e99f6b2b8a76/mvvm/src/main/java/com/thefuntasty/mvvm/delegate/ViewModelFragmentDelegate.kt#L14"></span>Functions with exact one statement, the return statement, can be rewritten with ExpressionBodySyntax.</td>
    </tr>
  </tbody>
</table>